src/ingestion/schemas/infoDataSchema.ts

```diff
@@ -14,6 +14,9 @@ const basicInfoDataSchema = z
   })
   .describe('InfoDataSchema');
 
+export const infoDataSchemaArray = z.array(basicInfoDataSchema);
+
+export type InfoDataWithFile = z.infer<typeof basicInfoDataSchema> & { fileName: string };
```

Comment:
line 47
`const isPixelSizeValid = pixelSize >= pixelSizeRange.min && pixelSize <= pixelSizeRange.max;`
Please use pixel size from mc-utils https://github.com/MapColonies/mc-utils/blob/master/src/geo/tiles.ts#L8
and remove from config

Reply:
line 27, remove empty line 
![image](https://github.com/MapColonies/ingestion-trigger/assets/18442671/cf249005-9901-4b8f-914a-42ef3147c49a)

Reply:
done!